ios/Podfile.lock

```diff
@@ -238,7 +238,7 @@ PODS:
     - React
   - react-native-status (1.0.0):
     - React
-  - react-native-status-keycard (2.5.37):
+  - react-native-status-keycard (2.5.38):
```

Comment:
also needed?

Reply:
yes, this adds support for Xcode 14 and iOS 16. https://github.com/status-im/react-native-status-keycard/pull/46